### DISCUSSION TITLE
Product identifier is missing in config file for Yale YRD210

### DISCUSSION THREAD

**Discussion by PBphl** [zwave-js/zwave-js#6895] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?
The device is not being properly identified as a Yale YRD210 lock.  According to Homeseer support "The fingerprint for the lock needs to be added to the list of devices."

I have included a pdf file that shows what is being reported for the Product Manufacturer and Product Identifiers.
I have also included a screen shot of the zwaveJS search for Product Manufacturer: 0x0109.  There is no entry for the Product Identifier reported by the YALE YRD210 lock with is 0x0004-0xffff.

  2. Click on '...'
  3. Scroll down to '...'

Anything else we should know?

[Screen captures.pdf](https://github.com/user-attachments/files/15597002/Screen.captures.pdf)

### Software versions

Driver (node-zwave-js): 12.9.1

Z-Wave JS UI: 9.13.3

Home Assistant Z-Wave Integration: ...

Home Assistant Z-Wave JS Addon: ...

ioBroker.zwave2 Adapter: ...

If you are using something non-standard, tell us here: ...

### Device information

Manufacturer: Yale
Model name: YRD210
Node ID: ...

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [ ] The log includes a re-interview of the problematic device _(if applicable)_.

- [ ] The log includes the problematic interaction with the device _(if applicable)_.

- [ ] I provided the node ID of the problematic device _(if applicable)_.

[a comment by an automated account is not shown]

**Comment by AlCalzone** [comment, marked as answer]
Escalated to an issue: https://github.com/zwave-js/node-zwave-js/issues/6898
